cves/kernel/CVE-2021-38166.yml.bak

```diff
@@ -219,7 +218,7 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
+  name: bpf
   note:
```

Comment:
You might want to include details about what this subsystem is or what its purpose is.